src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Options/ProcessorOptions.cs

```diff
@@ -0,0 +1,12 @@
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Options;
+
+public class ProcessorOptions
+{
+    public static readonly string Section = "Processor";
+
+    /// <summary>
+    /// Batch size to use when inserting location option meta rows, location option meta link rows,
+    /// and filter option meta link rows into the public data db.
+    /// </summary>
+    public required int InsertBatchSize { get; set; }
```

Comment:
Might want to name this more explicitly to mention meta i.e. `MetaInsertBatchSize`, otherwise, it sounds a bit too generic and could apply in many other contexts.